client/src/App.js

```diff
@@ -1,13 +1,18 @@
+import React from "react";
 import "./App.css";
+import MainContainer from "./components/MainContainer";
+import youtubeLogo from "./components/images/youtubeLogo.png";
 
-function App() {
+const App = () => {
   return (
-    <div className="App">
-      <header className="App-header">
-        <h1>Video Recommendation</h1>
-      </header>
+    <div className="App text-left">
```

Comment:
If you are not using the `.App` as a CSS class, feel free to remove it. It's often best to de-clutter our codebase as much as possible.

Reply:
I have removed the import related to CSS and also removed App.css file as I am not using CSS.